test/empty_manifold.jl

```diff
@@ -23,42 +28,42 @@ struct NonManifold <: Manifold end
 
     exp_retr = ManifoldsBase.ExponentialRetraction()
 
-    @test_throws ErrorException retract!(m, [0], [0], [0])
```

Comment:
Why did you replace these tests with `MPoint`-based ones instead of keeping both?

Reply:
Oh, indeed, one can even keep both – also a good idea. There was no specific reason to “delete”, I merely edited them. I will bring the old ones back, too.